cpp/ps0000/ps0000_slon.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream> // 1/2
+
+int main(void) // 1/1
+{
+  std::cout << "Hello, World!\n"; // 3/4 
+  return 0; // 1/1
+}
+// good 7/10 ✌
```

Comment:
- [x] most of code errors and bugs are fixed
- [ ] file is in the right path
- [x] +1 bonus marks for your first commit